Build/Build.cs

```diff
@@ -286,11 +286,16 @@ class Build : NukeBuild
                 customLogger: (_, msg) => Error(msg));
         });
 
-    bool HasDocumentationChanges =>
-        Changes.Any(x => x.StartsWith("docs"));
+    bool HasDocumentationChanges => Changes.Any(x => IsDocumentation(x));
 
-    bool HasSourceChanges =>
-        Changes.Any(x => !x.StartsWith("docs"));
+    bool HasSourceChanges => Changes.Any(x => !IsDocumentation(x));
+
+    static bool IsDocumentation(string x) =>
+        x.StartsWith("docs") ||
+        x.StartsWith("CONTRIBUTING.md") ||
+        x.StartsWith("cSpell.json") ||
+        x.StartsWith("LICENSE") ||
+        x.StartsWith("README.md");
```

Comment:
Gave the repo another look, and [`package.json`](https://github.com/fluentassertions/fluentassertions/blob/develop/package.json) should also be added here.

Reply:
ok